.github/labeler.yml

```diff
@@ -0,0 +1,7 @@
+# PRs that only touch the docs folder
+documentation:
+- all: ["docs/**/*"]
+
+# PRs that only touch type files
+typescript:
+- all: ["**/types/**/*"]
```

Comment:
miss fastify.d.ts

Reply:
Addressed in https://github.com/fastify/fastify/pull/2908/commits/684727c550c10ebf27be0bde36518dd3edcf55ea by using `"**/*[.|-]d.ts"`.

This was the simplest approach I found.

For example, something like:
```
typescript:
- any: ["**/types/**/*"]
- any: ["**/*.d.ts"]
- any: ["**/*-d.ts"]
```
would need another `all` condition to exclude any other files and folders.